src/main/java/edu/ucla/library/iiif/fester/utils/ManifestLabelComparator.java

```diff
@@ -0,0 +1,51 @@
+package edu.ucla.library.iiif.fester.utils;
+
+import java.util.Comparator;
+import java.util.Iterator;
+
+import info.freelibrary.iiif.presentation.Collection;
+import info.freelibrary.iiif.presentation.properties.Value;
+
+import se.sawano.java.text.AlphanumericComparator;
+
+/**
+ * Comparator for sorting manifests alpha-numerically based on their label.
+ */
+public class ManifestLabelComparator implements Comparator<Collection.Manifest> {
+
+    private final AlphanumericComparator myComparator = new AlphanumericComparator();
+
+    /**
+     * Creates an Manifest Label comparator.
+     */
+    public ManifestLabelComparator() {
+    }
+
+    @Override
+    public int compare(final Collection.Manifest aFirstManifest, final Collection.Manifest aSecondManifest) {
+        // Each label is a list of values
+        final Iterator<Value> firstLabelValues = aFirstManifest.getLabel().getValues().iterator();
```

Comment:
Hmm, so this assumes the order is set with the most important label first? The ticket just seems to assume maybe that there is only one label or that maybe the most important is first? I wonder what happens when we have multiple labels from different languages (each saying the same thing in a different language, like we might have with the Sinai manifests). I wonder if we should clarify with Dawn whether first label is the most appropriate label or whether we'd want to pass a configurable language code into the comparator so the sorting could be done in a language-preferred way?

Reply:
This is a good point. I made some assumptions here that I shouldn't have. I can follow up with Dawn on this.